src/main.cc

```diff
@@ -178,6 +178,35 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
+signal_sigint_cb(ev_loop *loop, struct ev_signal *w, int revents)
+{
+	(void) loop;
+	(void) w;
+	(void) revents;
+
+	/**
+	 * If running in daemon mode, tarantool exits on SIGINT
+	 */
+	if (pid_file)
+		tarantool_exit(0);
+
+	/**
+ 	 * Setting prompt explicitly every time, cause of
+ 	 * need to return from search mode
+ 	 */
+	rl_set_prompt("tarantool> ");
+
+	ssize_t res = write(STDOUT_FILENO, "^C\n", sizeof("^C\n")-1);
+	if (res < 0)
+		tarantool_exit(errno);
```

Comment:
I doubt that it is so important to exit if we can't write `"^C\n"`. I would ignore it (with `(void)rc;`). BTW, we usually call such return code as `rc`.

Reply:
Done.